releasetasks/util.py

```diff
@@ -52,3 +54,11 @@ def buildbot2ftp(platform):
 
 def buildbot2bouncer(platform):
     return bouncer_platform_map.get(platform, platform)
+
+
+def get_json_rev(repo_path, revision):
+    url = "https://hg.mozilla.org/{repo_path}/json-rev/{revision}".format(
```

Comment:
Can we use `redo` here incase hg is temporarily down, we don't need a long retry duration, just a max of 2 min with ramp would work for me